code/modules/research/designs/wiremod_designs.dm

```diff
@@ -467,6 +467,26 @@
 	id = "comp_assoc_list_pick"
 	build_path = /obj/item/circuit_component/list_pick/assoc
 
+/datum/design/component/bci/bci_camera
+	name = "BCI Camera"
+	id = "comp_camera_bci"
+	build_path = /obj/item/circuit_component/compare/remotecam/bci
+
+/datum/design/component/drone_camera
+	name = "Drone Camera"
+	id = "comp_camera_drone"
+	build_path = /obj/item/circuit_component/compare/remotecam/drone
+
+/datum/design/component/airlock_camera
+	name = "Airlock Camera"
+	id = "comp_camera_airlock"
+	build_path = /obj/item/circuit_component/compare/remotecam/airlock
+
+/datum/design/component/polaroid_camera
+	name = "Polaroid Camera Add-On"
+	id = "comp_camera_polaroid"
+	build_path = /obj/item/circuit_component/compare/remotecam/polaroid
```

Comment:
Change these to be from within the shells instead of being printable components from the component printer.

Reply:
BCI can probably be the exception though